Header header

string[] name
string[] talon_mode
string[] demand1_type
float64[] demand1_value
float64[] position
float64[] speed
float64[] output_voltage
float64[] output_current
float64[] bus_voltage
float64[] motor_output_percent
float64[] temperature
string[]  feedback_sensor
float64[] feedback_coefficient
string[]  remote_feedback_sensor
int8[]    remote_feedback_device_id0
string[]  remote_feedback_filter0
int8[]    remote_feedback_device_id1
string[]  remote_feedback_filter1
string[]  sensor_term_sum0
string[]  sensor_term_sum1
string[]  sensor_term_diff0
string[]  sensor_term_diff1
int32[]   encoder_ticks_per_rotation
int32[]   pid_slot
float64[] pid_p0
float64[] pid_i0
float64[] pid_d0
float64[] pid_f0
int32[] pid_izone0
int32[] pid_allowable_closed_loop_error0
float64[] pid_max_integral_accumulator0
float64[] pid_closed_loop_peak_output0
int32[]   pid_closed_loop_period0
float64[] pid_p1
float64[] pid_i1
float64[] pid_d1
float64[] pid_f1
int32[] pid_izone1
int32[] pid_allowable_closed_loop_error1
float64[] pid_max_integral_accumulator1
float64[] pid_closed_loop_peak_output1
int32[]   pid_closed_loop_period1
float64[] pid_p2
float64[] pid_i2
float64[] pid_d2
float64[] pid_f2
int32[] pid_izone2
int32[] pid_allowable_closed_loop_error2
float64[] pid_max_integral_accumulator2
float64[] pid_closed_loop_peak_output2
int32[]   pid_closed_loop_period2
float64[] pid_p3
float64[] pid_i3
float64[] pid_d3
float64[] pid_f3
int32[] pid_izone3
int32[] pid_allowable_closed_loop_error3
float64[] pid_max_integral_accumulator3
float64[] pid_closed_loop_peak_output3
int32[]   pid_closed_loop_period3
bool[]    aux_pid_polarity
float64[] set_point
int8[] can_id
float64[] closed_loop_error
float64[] integral_accumulator
float64[] error_derivative
float64[] closed_loop_target
float64[] p_term
float64[] i_term
float64[] d_term
float64[] f_term
float64[] active_trajectory_position
float64[] active_trajectory_velocity
float64[] active_trajectory_arbitrary_feed_forward
float64[] active_trajectory_heading
bool[] forward_limit_switch
bool[] reverse_limit_switch
bool[] forward_softlimit
bool[] reverse_softlimit
bool[] invert
bool[] sensorPhase
string[] neutral_mode
bool[] neutral_output
float64[] closed_loop_ramp
float64[] open_loop_ramp
float64[] peak_output_forward
float64[] peak_output_reverse
float64[] nominal_output_forward
float64[] nominal_output_reverse
float64[] neutral_deadband
float64[] voltage_compensation_saturation
int32[]   voltage_measurement_filter
bool[]    voltage_compensation_enable
int32[]   velocity_measurement_period
int32[]   velocity_measurement_window
string[]  limit_switch_local_forward_source
string[]  limit_switch_local_forward_normal
string[]  limit_switch_local_reverse_source
string[]  limit_switch_local_reverse_normal
string[]  limit_switch_remote_forward_source
string[]  limit_switch_remote_forward_normal
uint8[]   limit_switch_remote_forward_id
string[]  limit_switch_remote_reverse_source
string[]  limit_switch_remote_reverse_normal
uint8[]   limit_switch_remote_reverse_id
float64[] softlimit_forward_threshold
bool[]    softlimit_forward_enable
float64[] softlimit_reverse_threshold
bool[]    softlimit_reverse_enable
bool[]    softlimits_override_enable
int32[]   current_limit_peak_amps
int32[]   current_limit_peak_msec
int32[]   current_limit_continuous_amps
bool[]    current_limit_enable
float64[] motion_cruise_velocity
float64[] motion_acceleration
uint8[]   motion_s_curve_strength
int32[]   motion_profile_top_level_buffer_count
bool[]    motion_profile_top_level_buffer_full
int32[]   motion_profile_status_top_buffer_rem
int32[]   motion_profile_status_top_buffer_cnt
int32[]   motion_profile_status_btm_buffer_cnt
bool[]    motion_profile_status_has_underrun
bool[]    motion_profile_status_is_underrun
bool[]    motion_profile_status_active_point_valid
bool[]    motion_profile_status_is_last
int32[]   motion_profile_status_profile_slot_select0
int32[]   motion_profile_status_profile_slot_select1
string[]  motion_profile_status_output_enable
int32[]   motion_profile_time_dur_ms
uint8[]   status_1_general_period
uint8[]   status_2_feedback0_period
uint8[]   status_3_quadrature_period
uint8[]   status_4_aintempvbat_period
uint8[]   status_6_misc_period
uint8[]   status_7_commstatus_period
uint8[]   status_8_pulsewidth_period
uint8[]   status_9_motprofbuffer_period
uint8[]   status_10_motionmagic_period
uint8[]   status_11_uartgadgeteer_period
uint8[]   status_12_feedback1_period
uint8[]   status_13_base_pidf0_period
uint8[]   status_14_turn_pidf1_period
uint8[]   status_15_firmwareapistatus_period
uint8[]   control_3_general_period
uint8[]   control_4_advanced_period
uint8[]   control_5_feedbackoutputoverride_period
uint8[]   control_6_motprofaddtrajpoint_period
float64[] motion_profile_trajectory_period
string[]  faults
string[]  sticky_faults
float64[] conversion_factor
talon_state_controller/CustomProfileStatus[] custom_profile_status
bool[]    water_game

